### PR TITLE
New clean config structure with packages

### PR DESCRIPTION
**Pull request by Schluggi**
As described in #56 i changed the whole config to a more general setup.

Now you only need the `esphome.yaml` file. No duplications of anymore.
Please test it and give feedback :)

### CODE REVIEW

**Comment by cadriel** [comment]
One thing you might want to consider is using the new `friendly_name` property in ESPHome. If defined, it automatically prefixes all of your entities with the friendly name - meaning you don't have to substitute that in every sensor name. It also makes it much easier to have more than one AIOSense (or any other device for that matter..) in a single home.

For example, in my config - I have;

```
name: "SomeUniqueName"
friendly_name: "Office Sense"

sensor:
  - platform: bme680
    temperature:
      name: "Temperature"
      ...
```

Now, the device is called "Office Sense" in home assistant, and the sensor entity is automatically called "Office Sense Temperature".

It makes things much easier!

File: esphome/esphome.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  # ld2410_max_distance_still: 4.5m
+  # ld2410_timeout: 5s
+
+  # pir_delayed_off: 1s
+  # pir_delayed_on: 0s
+
+packages:
+  remote_package:
+    url: https://github.com/schluggi/AIOsense
+    ref: packages
```
Comment (MeisterGig) [inline esphome/esphome.yaml:22]: Der ref muss vor dem merge noch raus

Comment (Schluggi) [reply esphome/esphome.yaml:22]: Yes. But otherwise no one could test it yet :)

**Comment by Schluggi** [comment]
> One thing you might want to consider is using the new `friendly_name` property in ESPHome

@cadriel Thank you for the hint. I'll change it :)

**Comment by MeisterGig** [comment]
The occupancy,.yaml doesn't support the ld2410 yet. Maybe we could still add that feature :)

**Comment by Schluggi** [comment]
> The occupancy,.yaml doesn't support the ld2410 yet. Maybe we could still add that feature :)

Thanks. Should be fixed.

**Pull request merged**